cmd/serverless/main.go

```diff
@@ -0,0 +1,384 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	_ "expvar"
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+	"os/signal"
+	"runtime"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/spf13/cobra"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/dogstatsd"
+	"github.com/DataDog/datadog-agent/pkg/forwarder"
+	"github.com/DataDog/datadog-agent/pkg/serializer"
+	"github.com/DataDog/datadog-agent/pkg/serverless"
+	"github.com/DataDog/datadog-agent/pkg/util/flavor"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/datadog-agent/pkg/version"
+)
+
+var (
+	// serverlessAgentCmd is the root command
+	serverlessAgentCmd = &cobra.Command{
+		Use:   "agent [command]",
+		Short: "Serverless Datadog Agent at your service.",
+		Long: `
+Datadog Serverless Agent accepts custom application metrics points over UDP, aggregates and forwards them to Datadog,
+where they can be graphed on dashboards. The Datadog Serverless Agent implements the StatsD protocol, along with a few extensions for special Datadog features.`,
+	}
+
+	startCmd = &cobra.Command{
+		Use:   "start",
```

Comment:
Details, but we use `run` on the main agent (we deprecated `start` as it implies it would run the agent as a daemon).

Reply:
Renamed to `run` for consistency.